### ISSUE TITLE
Some issues with the installation environment

### ISSUE THREAD

**Issue by NanCheng2001** [cvlab-stonybrook/SAMPath#9]
When I run the following program：
`pip install monai torchmetrics==0.11.4 pytorch_lightning==2.0.2 albumentations box wandb`

**the error is as follows：**
```
ERROR: Could not find a version that satisfies the requirement box (from versions: none)
ERROR: No matching distribution found for box
```
```

It seems that the box library cannot be found. I couldn't find any information about the box library online either.

**Comment by jingweizhang-xyz** [comment]
Sorry, I believe it should be python-box. Check https://github.com/cdgriffith/Box for more information.

If it does not work, let me know.

**Comment by NanCheng2001** [comment]
OK, I will try it. I will inform you as soon as possible if it matches.

**Comment by NanCheng2001** [comment]
![image](https://github.com/cvlab-stonybrook/SAMPath/assets/96007479/d625a51f-2ac9-40bd-b62c-232e27e11758)
Hello! I encountered the error mentioned above when predicting the network. Can you help me answer this? Does it appear that config is not compatible with the format of MMEngine? Here is my BCSS.py config:
```
from box import Box

config = {
    "batch_size": 12,
    "accumulate_grad_batches": 2,
    "num_workers": 4,
    "out_dir": "/mnt/project/SAM/SAMPath/SAMPath/output",
    "opt": {
        "num_epochs": 32,
        "learning_rate": 1e-4,
        "weight_decay": 1e-2, #1e-2,
        "precision": 32, # "16-mixed"
        "steps":  [72 * 25, 72 * 29],
        "warmup_steps": 72,
    },
    "model": {
        "type": 'vit_b',
        "checkpoint": "/mnt/project/SAM/SAMPath/SAMPath/checkpoints/sam_vit_b_01ec64.pth",
        "freeze": {
            "image_encoder": True,
            "prompt_encoder": True,
            "mask_decoder": False,
        },
        "prompt_dim": 256,
        "prompt_decoder": False,
        "dense_prompt_decoder": False,

        "extra_encoder": 'hipt',
        "extra_type": "fusion",
        "extra_checkpoint":  "/mnt/project/SAM/SAMPath/SAMPath/checkpoints/vit256_small_dino.pth",
    },
    "loss": {
        "focal_cof": 0.25,
        "dice_cof": 0.75,
        "ce_cof": 0.0,
        "iou_cof": 0.0625,
    },
    "dataset": {
        "dataset_root": "/mnt/dataset/BCSS/merged_dataset",
        "dataset_csv_path": "/mnt/project/SAM/SAMPath/SAMPath/dataset_cfg/BCSS_cv.csv",
        "val_fold_id": 0,
        "num_classes": 6,

        "ignored_classes": (0),
        "ignored_classes_metric": None, # if we do not count background, set to 1 (bg class)
        "image_hw": (1024, 1024), # default is 1024, 1024

        "feature_input": False, # or "True" for *.pt features
        "dataset_mean": (0.485, 0.456, 0.406),
        "dataset_std": (0.229, 0.224, 0.225),
    }
}

cfg = Box(config)
```

**Comment by NanCheng2001** [comment]
![image](https://github.com/cvlab-stonybrook/SAMPath/assets/96007479/d7170000-ca70-4104-9113-47cc991b99e3)
The error position in preidct.py is here
